e2e/test/scenarios/dashboard-filters/reproductions/22788-flter-cc-dropped-on-second-edit.cy.spec.js

```diff
@@ -98,16 +97,11 @@ describe("issue 22788", () => {
 
 function addFilterAndAssert() {
   filterWidget().click();
-  popover().within(() => {
```

Comment:
Here and in another test I revert changes made by made previous PR. It turned out that MBQL lib matches fields incorrectly in this case.